platform/platform-app/pom.xml

```diff
@@ -423,22 +424,28 @@
         <cxf.abdera.osgi.version>1.1.3_2</cxf.abdera.osgi.version>
         <cxf.ahc.version>1.9.8</cxf.ahc.version>
         <cxf.atmosphere.version>2.3.7</cxf.atmosphere.version>
+        <cxf.brave.version>4.3.1</cxf.brave.version>
+        <cxf.brave.reporter.version>0.9.0</cxf.brave.reporter.version>
+        <cxf.brave.zipkin.version>1.24.0</cxf.brave.zipkin.version>
         <cxf.classmate.version>1.1.0</cxf.classmate.version>
         <cxf.commons-codec.version>${commons-codec-version}</cxf.commons-codec.version>
         <cxf.commons-collections.version>3.2.2</cxf.commons-collections.version>
         <cxf.commons-lang.version>2.6</cxf.commons-lang.version>
         <cxf.commons-lang3.version>3.5</cxf.commons-lang3.version>
         <cxf.dropwizard.version>3.1.2</cxf.dropwizard.version>
-        <cxf.ehcache.version>2.10.3</cxf.ehcache.version>
+        <cxf.ehcache.version>2.10.4</cxf.ehcache.version>
         <cxf.fastinfoset.bundle.version>1.2.13_1</cxf.fastinfoset.bundle.version>
-        <cxf.guava.version>18.0</cxf.guava.version>
+        <cxf.findbugs.version>3.0.2</cxf.findbugs.version>
+        <cxf.guava.version>${guava.version}</cxf.guava.version>
         <cxf.hazelcast.version>1.9.4.8</cxf.hazelcast.version>
         <cxf.httpcomponents.asyncclient.version>4.1.3</cxf.httpcomponents.asyncclient.version>
         <cxf.httpcomponents.client.version>4.5.3</cxf.httpcomponents.client.version>
         <cxf.httpcomponents.core.version>4.4.6</cxf.httpcomponents.core.version>
-        <cxf.jackson.version>2.8.7</cxf.jackson.version>
+        <cxf.jackson.version>2.8.8</cxf.jackson.version>
         <cxf.james.mim4j.version>0.7.2</cxf.james.mim4j.version>
         <cxf.javax.annotation-api.version>1.2</cxf.javax.annotation-api.version>
+        <cxf.json.api.version>1.0</cxf.json.api.version>
+        <cxf.johnzon.version>1.0.0</cxf.johnzon.version>
```

Comment:
Sorry. Just saw something else. A few of these properties that you added in this file aren't used at all in ddf/platform/platform-app/src/main/resources/features.xml, so they don't need to be included. The "DDF changes made" code comment includes "Removed properties that are not used".

Reply:
These must have been the properties that came from the cxf features that I had just removed. My mistake, I will remove the unused properties.

Reply:
Removed.